Plugins/Flow.Launcher.Plugin.BrowserBookmark/FirefoxBookmarkLoader.cs

```diff
@@ -5,140 +5,133 @@
+    public abstract List<Bookmark> GetBookmarks();
 
-        private const string queryAllBookmarks = @"SELECT moz_places.url, moz_bookmarks.title
-              FROM moz_places
-              INNER JOIN moz_bookmarks ON (
+    private const string QueryAllBookmarks = """
+        SELECT moz_places.url, moz_bookmarks.title
+        FROM moz_places
+            INNER JOIN moz_bookmarks ON (
                 moz_bookmarks.fk NOT NULL AND moz_bookmarks.title NOT NULL AND moz_bookmarks.fk = moz_places.id
-              )              
-              ORDER BY moz_places.visit_count DESC
-            ";
+            )
+        ORDER BY moz_places.visit_count DESC
+        """;
 
-        private const string dbPathFormat = "Data Source ={0}";
+    private const string DbPathFormat = "Data Source ={0}";
 
-        protected static List<Bookmark> GetBookmarksFromPath(string placesPath)
-        {
-            // Return empty list if the places.sqlite file cannot be found
-            if (string.IsNullOrEmpty(placesPath) || !File.Exists(placesPath))
-                return new List<Bookmark>();
-
-            var bookmarkList = new List<Bookmark>();
-
-            Main.RegisterBookmarkFile(placesPath);
-
-            // create the connection string and init the connection
-            string dbPath = string.Format(dbPathFormat, placesPath);
-            using var dbConnection = new SqliteConnection(dbPath);
-            // Open connection to the database file and execute the query
-            dbConnection.Open();
-            var reader = new SqliteCommand(queryAllBookmarks, dbConnection).ExecuteReader();
-            
-            // return results in List<Bookmark> format
-            bookmarkList = reader.Select(
-                x => new Bookmark(x["title"] is DBNull ? string.Empty : x["title"].ToString(),
-                    x["url"].ToString())
-            ).ToList();
-
-            return bookmarkList;
-        }
+    protected static List<Bookmark> GetBookmarksFromPath(string placesPath)
+    {
+        // Return empty list if the places.sqlite file cannot be found
+        if (string.IsNullOrEmpty(placesPath) || !File.Exists(placesPath))
+            return new List<Bookmark>();
+
+        Main.RegisterBookmarkFile(placesPath);
+
+        // create the connection string and init the connection
+        string dbPath = string.Format(DbPathFormat, placesPath);
+        using var dbConnection = new SqliteConnection(dbPath);
+        // Open connection to the database file and execute the query
+        dbConnection.Open();
+        var reader = new SqliteCommand(QueryAllBookmarks, dbConnection).ExecuteReader();
+
+        // return results in List<Bookmark> format
+        return reader
+            .Select(
+                x => new Bookmark(
+                    x["title"] is DBNull ? string.Empty : x["title"].ToString(),
+                    x["url"].ToString()
+                )
+            )
+            .ToList();
```

Comment:
I think it's much more readable when every method call in the chain is on its own line. I'd like to hear your opinion.

Reply:
Yeah I think it's fine. Though for me, I majorly like putting the first `.Select` on the same line...though I don't know how others will think about this. @Flow-Launcher/team